requirements-data_and_models.txt

```diff
@@ -25,3 +25,4 @@ https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_jnlpba
 matplotlib==3.4.1
 scipy==1.6.2
 transformers==4.5.1
+spacy_lookups_data==1.0.0
```

Comment:
I suppose this is for the lemmatizer look-up tables? Surprised to see it's 100MB heavy.

By the way, what's the role of the lemmatizer in our pipeline? Is it required by the transformers or the NER components? I'm just a bit out of the loop on that one.

Reply:
Yes.

Please have a look at the following two comments answering these questions:
- point (1) in https://github.com/BlueBrain/Search/pull/351#discussion_r634435602
- https://github.com/BlueBrain/Search/pull/351#discussion_r634368624

Reply:
👍🏻 So, short answer is the entity ruler needs lemmatized inputs, right?

Reply:
Yes!

Reply:
Resolving the discussion to make reading the reviews easier. Feel free to reopen if needed.